DisguiseUnityRenderStream/Runtime/DisguiseRemoteParameters.cs

```diff
@@ -292,16 +306,36 @@ public List<ManagedRemoteParameter> exposedParameters()
         }
         return parameters;
     }
+    
+    MemberInfo GetMemberInfo(UnityEditor.SerializedProperty property, out bool isPublic)
```

Comment:
Minor / Opinion: I must admit I was a little bit confused of this method having the same name as the other public one but have different roles.